src/main/java/pl/schoolmanager/entity/School.java

```diff
@@ -112,5 +118,32 @@ public void setStudent(Set<Student> student) {
 	public String getNameForForm() {
 		return this.name + " (" +this.type +")";
 	}
+
+	public TimeTable getTimeTable() {
+		return timeTable;
+	}
+
+	public void setTimeTable(TimeTable timeTable) {
+		this.timeTable = timeTable;
+	}
+	
+	public List<String> getTimeTableList() {
+		TimeTable tt = this.timeTable;
+		if (tt == null) {
+			tt = new TimeTable();
```

Comment:
it can be shorter:
if(timeTable == null) {
   timeTable = new TimeTable();
}

Reply:
Thanks. To be discussed.